.github/workflows/linters.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       - name: Setup Webhint
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/223380131-9995012c-82d6-4b17-bfcd-355d83c0a8ee.png)

- [ ] Kindly remove those links which are not working anymore to make your readme file descriptive and professional :blush: